docs/sources/installation/mac.md

```diff
@@ -63,19 +64,19 @@ Then start Grafana using:
 brew services start grafana
 ```
 
-Default login and password `admin`/ `admin`
+Refer to [Getting started](docs\sources\guides\getting_started.md) for instructions on logging in for the first time and adding a data source.
 
-## Install from binary tar file
+## Install from binary .tar file
```

Comment:
See earlier question: Is this section relevant anymore? The link to grafana.com/get leads to either the Cloud install or back to the Downloads page, where there is no .tar.gz file option.

Reply:
@DanCech ?

Reply:
@DanCech, need an answer on this.

Reply:
I deleted the binary .tar since it is not available on the Mac page.